haze/src/commonMain/kotlin/dev/chrisbanes/haze/Haze.kt

```diff
@@ -149,18 +143,21 @@ object HazeDefaults {
     tint: Color = tint(backgroundColor),
     blurRadius: Dp = this.blurRadius,
     noiseFactor: Float = this.noiseFactor,
+    dropShadowSize: Float = this.dropShadowSize,
```

Comment:
Please remove the drop shadow work from this PR. We're missing Android support which needs to be added for it to land. Easier to remove and follow up